core/src/main/java/org/frankframework/receivers/Receiver.java

```diff
@@ -1335,6 +1337,7 @@ private Message processMessageInAdapter(MessageWrapper<M> messageWrapperOriginal
 					if (Message.isEmpty(pipeLineResult.getResult())) {
 						pipeLineResult.setResult(adapter.formatErrorMessage(null, t, compactedMessage, session, null));
 					}
+					session.setExitState(pipeLineResult);
```

Comment:
We did this for the "normal" exit from pipeline, but when adding return of formatted error-message I overlooked that this also needs to be done for the new error-return.